src/test/resources/cassandra-diagnostics.yml

```diff
@@ -0,0 +1,12 @@
+# Slow query threshold
+slowQueryThresholdInMilliseconds: 25
+
+# Log all queries or just slow queries
+logAllQueries: false
+
+# Reporters
+reporters:
+  - reporter: io.smartcat.cassandra.diagnostics.report.LogQueryReporter
+  - reporter: io.smartcat.cassandra.diagnostics.report.RiemannQueryReporter
```

Comment:
Can you add this changes in README as example, since now we should explain that you can use multiple reporters